lib/rack/mock.rb

```diff
@@ -254,9 +254,9 @@ def identify_cookie_attributes(cookie_filling)
       cookie_bits = cookie_filling.split(';')
       cookie_attributes = Hash.new
       cookie_attributes.store('value', cookie_bits[0].strip)
-      cookie_bits.each do |bit|
+      cookie_bits[1..].each do |bit|
```

Comment:
```suggestion
      cookie_bits.drop(1).each do |bit|
```

I don't think endless ranges work on Ruby 2.5, but `drop(1)` should do the same thing.